packages/stores/types.ts

```diff
@@ -1,5 +1,7 @@
+import { TPortfolioPos } from '@deriv/shared';
 import type { GetAccountStatus, Authorize, DetailsOfEachMT5Loginid } from '@deriv/api-types';
 import type { RouteComponentProps } from 'react-router';
+import { TContractInfo } from '../shared/src/utils/contract/contract-types';
```

Comment:
@sergei-deriv This package shouldn’t have dependencies on other packages just to reuse some types, You should move `TPortfolioPos` and `TContractInfo` to this package at least for now.
By the way can’t we use `PortfolioResponse` from `@deriv/api-types`? 🤔

Reply:
> @sergei-deriv This package shouldn’t have dependencies on other packages just to reuse some types, You should move `TPortfolioPos` and `TContractInfo` to this package at least for now. By the way can’t we use `PortfolioResponse` from `@deriv/api-types`? 🤔

I changed TPortfolioPos to PortfolioResponse.